### DISCUSSION TITLE
Default Filter includes retired devices

### DISCUSSION THREAD

**Discussion by ColtMercer** [nautobot/nautobot#3775] [Ideas]
Retired devices should be filtered out of views by default. Preferably configurable to multiple status fields. The home page counts includes retired devices and makes the inventory numbers inaccurate. We can't delete Retired devices because we have to keep the data for legal Records Management purposes.

**Comment by jvanderaa** [comment]
And it is also something that may need to be accounted for as an inventory source for automation as well. That today it would be recommended that you need to add in a `status=` on each of the inventory queries. Immediate thoughts includes being able to filter or a separate container (or app) that can move things into that would represent that retired information?
